### PR TITLE
Fhir security

### PR DESCRIPTION
**Pull request by edarchis**
Switch FHIR API to OpenIMIS authorization system

### CODE REVIEW

**Comment by edarchis** [comment]
I'd say that codeclimate is wrong here. It's basically configuration of the security, of course it looks repetitive. I could extract that into a different configuration and then try to dynamically bind these together but it seems much clearer this way.

File: api_fhir/views.py [hunk 1]
```diff
@@ -35,14 +42,14 @@ class BaseFHIRView(APIView):
 
 class InsureeViewSet(BaseFHIRView, viewsets.ModelViewSet):
     lookup_field = 'uuid'
-    queryset = Insuree.objects.all()
     serializer_class = PatientSerializer
+    permission_classes = (FHIRApiInsureePermissions,)
 
     def list(self, request, *args, **kwargs):
         claim_date = request.GET.get('claimDateFrom')
         identifier = request.GET.get("identifier")
```
Comment (dragos-dobre) [inline api_fhir/views.py:44]: We should uniformize the quotes.

File: api_fhir/permissions.py [hunk 2]
```diff
@@ -1,17 +1,83 @@
+from claim import ClaimConfig
```
Comment (dragos-dobre) [inline api_fhir/permissions.py]: Is there a difference as here is claim and all the others have '.apps' in module name?

**Review by dragos-dobre — APPROVED** [review]

File: api_fhir/permissions.py (hunk 2 above)
Comment (xgill) [reply api_fhir/permissions.py]: the __init__.py in claim makes it work, but indeed, better be homogenous,
Thx

**Pull request merged**